NAMESPACE

```diff
@@ -6,7 +6,6 @@ export(cookie_server)
 export(cookie_ui)
 export(logout_server)
 export(logout_ui)
-import(config)
```

Comment:
@dietrichson Why are you removing the config package ? This is used by get_config().

Reply:
By mistake. I commented out the @import statements as well. I attempted to correct this in https://github.com/chi2labs/cognitoR/pull/10/commits/4c8ec41c6529559b2c9c00f040e604e9c8a76d51